src/RuntimeStoreVerifier/RuntimeStoreVerifierCommand.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.RegularExpressions;
+using Microsoft.Extensions.CommandLineUtils;
+using WinTrustSources;
+
+namespace RuntimeStoreVerifier
+{
+    public class RuntimeStoreVerifierCommand
+    {
+        internal static int Execute(
+            CommandOption exclusionFileOption,
+            CommandOption verboseOption,
+            CommandArgument pathArgument)
+        {
+            var exclusions = exclusionFileOption.HasValue() ? File.ReadAllLines(exclusionFileOption.Value()) : null;
```

Comment:
Any reason this tool can't be a task instead? MSBuild item groups already have powerful include/exclude filtering.